admin/src/views/LiveDesk.vue

```diff
@@ -226,7 +226,7 @@ export default Vue.extend({
     async reset() {
       const admin = firebase.auth().currentUser!.email;
       const ref = db
-        .collection('DH6')
+        .collection('DH7')
```

Comment:
the reason why is because im not sure how to test LiveDesk as i cant get into this component. can you assist me with that? i just gotta make sure the store is avail in that component